### ISSUE TITLE
LSP not found

### ISSUE THREAD

**Issue by moooooo16** [linux-cultist/venv-selector.nvim#181]
Hi,

Im using jedi as LSP, anaconda as env manager. It can search the location correct (I can select them use venv-selector) but It shows "No python lsp servers are running. Please open a python file and them select a venv to activate.

Below is my code, I read through README file in this repo, but it seems didnt mention about this issue.
`local options = {
    settings = {
        search = {
            anaconda_base = {
                command = "fd --full-path 'bin/python$' /opt/homebrew/Caskroom/miniconda/base/envs",
                type = "anaconda",
            },
        },
    },
}

require("venv-selector").setup(options)`

**Comment by akthe-at** [comment]
@moooooo16 Are you in an open python file when you try do do this?

**Comment by moooooo16** [comment]
> @moooooo16 Are you in an open python file when you try do do this? 

Yes, I opened python file first and then the plugin.

**Comment by moooooo16** [comment]
nvm, I change to pyright, and everything work as charm!

**Comment by linux-cultist** [comment]
Yeah jedi is very untested at this point... May be as simple as just adding it to the code but haven't tried it. :)

**Comment by lazyzyf** [comment]
i use ruff server native lsp, venv-selector also report the same error,please help to fix this.
